### PR TITLE
Corrections for ShowInterfacesSwitchport Class

### PR DESCRIPTION
**Pull request by jdrew82**
Corrected regex for VLAN names to allow special characters and save 
matched name in ShowInterfacesSwitchport class

Added handling for invisible characters at front of string for
ShowInterfacesSwitchport class

### CODE REVIEW

**Comment by jdrew82** [comment]
I found that your previous regex wasn't matching on the Access VLAN portion due to our use of special characters that aren't matched with \w. Using a .* allows it to match anything in the parenthesis. Also, for some reason there are invisible characters at the start of the string that are causing the line to not match. They appear to be '--More-- \x08\x08\x08\x08\x08\x08\x08\x08\x08        \x08\x08\x08\x08\x08\x08\x08\x08\x08' which breaks the parsing.

**Comment by jeaubin** [comment]
@jdrew82  - Cool ! thanks! 

We will look into it and merge it next week.

File: src/genie/libs/parser/iosxe/show_interface.py [hunk 1]
```diff
@@ -1333,6 +1335,7 @@ def cli(self, interface='', output=None):
             m = p8.match(line)
             if m:
                 ret_dict[intf]['access_vlan'] = m.groupdict()['access_vlan']
+                ret_dict[intf]['access_vlan_name'] = m.groupdict()['access_vlan_name']
```
Comment (lubnarasheed) [inline src/genie/libs/parser/iosxe/show_interface.py]: It seems like the regex key name is `dummy`, please update the regex as well to have the key as `access_vlan_name`

https://github.com/CiscoTestAutomation/genieparser/blob/master/src/genie/libs/parser/iosxe/show_interface.py#L1332

File: src/genie/libs/parser/iosxe/show_interface.py [hunk 2]
```diff
@@ -1343,6 +1346,7 @@ def cli(self, interface='', output=None):
                 if 'encapsulation' not in ret_dict[intf]:
                     ret_dict[intf]['encapsulation'] = {}
                 ret_dict[intf]['encapsulation']['native_vlan'] = m.groupdict()['native_vlan']
+                ret_dict[intf]['encapsulation']['native_vlan_name'] = m.groupdict()['native_vlan_name']
```
Comment (lubnarasheed) [inline src/genie/libs/parser/iosxe/show_interface.py]: and the sam thing here about the key being `dummy` and not `native_vlan_name`
https://github.com/CiscoTestAutomation/genieparser/blob/master/src/genie/libs/parser/iosxe/show_interface.py#L1340

File: src/genie/libs/parser/iosxe/show_interface.py (hunk 1 above)
Comment (jdrew82) [reply src/genie/libs/parser/iosxe/show_interface.py]: Strange, thought I had already updated that. Must have been missed in my commit.

File: src/genie/libs/parser/iosxe/show_interface.py (hunk 2 above)
Comment (jdrew82) [reply src/genie/libs/parser/iosxe/show_interface.py]: Must have been missed in my commit. Will get corrected.

Comment (lubnarasheed) [reply src/genie/libs/parser/iosxe/show_interface.py]: great! saw that the sample output line is already in

File: src/genie/libs/parser/iosxe/show_interface.py (hunk 1 above)
Comment (lubnarasheed) [reply src/genie/libs/parser/iosxe/show_interface.py]: great!

File: src/genie/libs/parser/iosxe/show_interface.py (hunk 2 above)
Comment (jdrew82) [reply src/genie/libs/parser/iosxe/show_interface.py]: Should be corrected now. Apologies about that.

File: src/genie/libs/parser/iosxe/show_interface.py (hunk 1 above)
Comment (jdrew82) [reply src/genie/libs/parser/iosxe/show_interface.py]: Should be corrected now. Apologies for that.

**Comment by jdrew82** [comment]
I noticed that my other change for the regex somehow was missed in the commit so I've re-added it. The \w+ in the regex doesn't allow for use of : or _ which we use. Changing to .* allows for any characters in the ( ).

File: src/genie/libs/parser/iosxe/show_interface.py (hunk 1 above)
Comment (lubnarasheed) [reply src/genie/libs/parser/iosxe/show_interface.py]: Np, looks good now

File: src/genie/libs/parser/iosxe/show_interface.py (hunk 2 above)
Comment (lubnarasheed) [reply src/genie/libs/parser/iosxe/show_interface.py]: Np, looks good now, thanks!

**Comment by lubnarasheed** [comment]
@jdrew82 thanks for enhancing this! It will be part of the next release

**Pull request merged**